adapters/repos/db/index.go

```diff
@@ -607,10 +607,10 @@ func (i *Index) objectByID(ctx context.Context, id strfmt.UUID,
 			}
 		}
 		if replProps.NodeName != "" {
-			obj, err = i.replicator.NodeObject(ctx, replProps.NodeName, shardName, id, props, addl)
+			obj, err = i.replicator.NodeObject(ctx, replProps.NodeName, shardName, id)
 		} else {
 			obj, err = i.replicator.GetOne(ctx,
-				replica.ConsistencyLevel(replProps.ConsistencyLevel), shardName, id, props, addl)
```

Comment:
We to pass `props, addl` to `replicator.GetOne` since they are part of original request

Reply:
I removed this because if you trace these params to `shard.objectByID`, they are never used. But I can add this back if you like